### ISSUE TITLE
Missing sklearn module dependency

### ISSUE THREAD

**Issue by krcroft** [bukosabino/ta#45]
`pip3 install ta` succeeds.

```
>>> import pandas as pd
>>> from ta import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/ta/__init__.py", line 11, in <module>
    from .pipeline_wrapper import *
  File "/usr/local/lib/python3.6/dist-packages/ta/pipeline_wrapper.py", line 2, in <module>
    from sklearn.base import TransformerMixin, BaseEstimator
ModuleNotFoundError: No module named 'sklearn'
```
So it looks like the 'ta' package is missing sklearn as a dependency.

`pip3 install sklearn` 

```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from ta import *
>>> 
```

**Comment by bukosabino** [comment]
Fixed on new v0.4.0 release!

**Comment by krcroft** [comment]
Thank you @bukosabino !
